src/main/scala/za/co/absa/hyperdrive/trigger/configuration/application/NotNullValidationException.scala

```diff
@@ -0,0 +1,28 @@
+
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hyperdrive.trigger.configuration.application
+
+import org.springframework.boot.context.properties.source.ConfigurationPropertyName
+import za.co.absa.hyperdrive.trigger.configuration.application.NotNullValidationException.createMessage
+
+class NotNullValidationException(name: ConfigurationPropertyName) extends RuntimeException(createMessage(name))
+
+object NotNullValidationException {
+  def createMessage(name: ConfigurationPropertyName): String = {
+    s"${name} expected to be not null, but was null"
```

Comment:
I would say here property was not defined or something like that

Reply:
ok